Bronco_Tree/tree.pde

```diff
@@ -83,7 +93,7 @@ class Tree {
     // check all leaves
     for (int i = leaves.size()-1; i >= 0; i--) {
       // if a leaf is reached by a branch, remove it from further consideration
-      if (leaves.get(i).reached) {
+      if (leaves.get(i).bloomed || leaves.get(i).shouldExpire ) {  // added expiration to removal check
```

Comment:
Okay this clears up my comment above. Is this working currently? It seems like the leaves stay rendered even if they are expired/removed here